packages/office-ui-fabric-react/src/components/pickers/examples/TagPicker.Basic.Example.tsx

```diff
@@ -28,104 +40,63 @@ const _testTags: ITag[] = [
-          getTextFromItem={this._getTextFromItem}
-          pickerSuggestionsProps={{
-            suggestionsHeaderText: 'Suggested Tags',
-            noResultsFoundText: 'No Color Tags Found',
-          }}
-          itemLimit={2}
-          disabled={this.state.isPickerDisabled}
-          inputProps={{
-            onBlur: (ev: React.FocusEvent<HTMLInputElement>) => console.log('onBlur called'),
-            onFocus: (ev: React.FocusEvent<HTMLInputElement>) => console.log('onFocus called'),
-            'aria-label': 'Tag Picker',
-          }}
-        />
-      </div>
-    );
-  }
-
-  private _getTextFromItem(item: ITag): string {
-    return item.name;
-  }
+  const picker = React.useRef<IBasePicker<ITag>>(null);
+  const [tagPicker, { toggle: toggleTagPicker }] = useBoolean(false);
 
-  private _onDisabledButtonClick = (): void => {
-    this.setState({
-      isPickerDisabled: !this.state.isPickerDisabled,
-    });
-  };
+  const listContainsDocument = useConstCallback((tag: ITag, tagList?: ITag[]) => {
+    if (!tagList || !tagList.length || tagList.length === 0) {
+      return false;
+    }
+    return tagList.some(compareTag => compareTag.key === tag.key);
+  });
 
-  private _onFilterChanged = (filterText: string, tagList: ITag[]): ITag[] => {
+  const filterSuggestedTags = useConstCallback((filterText: string, tagList: ITag[]): ITag[] => {
     return filterText
-      ? _testTags
-          .filter(tag => tag.name.toLowerCase().indexOf(filterText.toLowerCase()) === 0)
-          .filter(tag => !this._listContainsDocument(tag, tagList))
+      ? testTags.filter(
+          tag => tag.name.toLowerCase().indexOf(filterText.toLowerCase()) === 0 && !listContainsDocument(tag, tagList),
+        )
       : [];
-  };
+  });
 
-  private _onFilterChangedNoFilter = (filterText: string, tagList: ITag[]): ITag[] => {
-    return filterText ? _testTags.filter(tag => tag.name.toLowerCase().indexOf(filterText.toLowerCase()) === 0) : [];
-  };
+  const filterSelectedTags = useConstCallback((filterText: string, tagList: ITag[]): ITag[] => {
+    return filterText ? testTags.filter(tag => tag.name.toLowerCase().indexOf(filterText.toLowerCase()) === 0) : [];
+  });
 
-  private _onItemSelected = (item: ITag): ITag | null => {
-    if (this._picker.current && this._listContainsDocument(item, this._picker.current.items)) {
+  const onItemSelected = useConstCallback((item: ITag): ITag | null => {
+    if (picker.current && listContainsDocument(item, picker.current.items)) {
       return null;
     }
     return item;
-  };
+  });
 
-  private _listContainsDocument(tag: ITag, tagList?: ITag[]) {
-    if (!tagList || !tagList.length || tagList.length === 0) {
-      return false;
-    }
-    return tagList.filter(compareTag => compareTag.key === tag.key).length > 0;
-  }
-}
+  return (
+    <div className={rootClass}>
+      <Checkbox styles={checkboxStyles} label="Disable Tag Picker" checked={tagPicker} onChange={toggleTagPicker} />
```

Comment:
Casing plus same point from earlier about toggles
```suggestion
      <Checkbox styles={checkboxStyles} label="Disable tag picker" checked={tagPicker} onChange={toggleTagPicker} />
```